src/sagas/templatesSaga.js

```diff
@@ -21,7 +21,7 @@ export function* pushTemplatesToStore() {
   try {
     const templateLibrary = new TemplateLibrary();
     const templateIndex = yield templateLibrary
-      .getTemplateIndex({ latestVersion: false, ciceroVersion });
+      .getTemplateIndex({ latestVersion: true, ciceroVersion });
```

Comment:
I'm not sure I understand this change. Do we want the latest version of the templates or the versions that match the cicero version provided? Will this only give us templates if the latest version matches the passed ciceroVersion?

Reply:
I am a little unclear on this. The new changes of `displayName` were not populating without this, but that was before I had updated `cicero-ui`.

I believe the `ciceroVersion` in `{ latestVersion: false, ciceroVersion }` filters for templates that match the cicero version provided from the node modules dependencies. Setting the `latestVersion` back to `false` results in two different versions of every template populating, as shown here: 

<img width="1680" alt="Screen Shot 2019-08-07 at 8 43 26 AM" src="https://user-images.githubusercontent.com/36460856/62623867-e0386e80-b8ef-11e9-87ab-7a23f53ab43c.png">

Reply:
Template Studio v1 uses: `templateLibrary.getTemplateIndex({ latestVersion: true, ciceroVersion });` and I thought it was for a good reason, but I can't recall what that was now...

Reply:
This works for me. I think it's an easy change in the future.